### PR TITLE
Added another 3rd party dep

### PR DESCRIPTION
**Pull request by tstavinoha**

### CODE REVIEW

**Comment by tstavinoha** [comment]
@googlebot I signed it!

File: third-party/README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ If you have written a useful extension that you think will benefit the Jib commu
 Interested in writing an extension? It's easy! Take a look at ["Writing Your Own Extensions"](../README.md#writing-your-own-extensions).
 
 - OSGi Bundle Packaging Plugin Extension ([Maven](https://github.com/thought-gang/jib-maven-plugin-extension.git)): an extension to containerize an OSGI bundle (Maven packaging type `bundle`)
+- Layer With Modification Time ([Maven](https://github.com/infobip/jib-layer-with-modification-time-extension-maven)): an extension that sets modification time for hosted web resources
```
Comment (chanseokoh) [inline third-party/README.md:10]: How about making it more descriptive?

```suggestion
- Layer With Modification Time ([Maven](https://github.com/infobip/jib-layer-with-modification-time-extension-maven)): an extension that sets modification time for hosted web resources, for example, selectively setting file timestamps to build time
```

**Review by chanseokoh — APPROVED** [review]
This is a cool extension! Thanks for writing and sharing it with the community!

**Review by chanseokoh — APPROVED** [review]

File: third-party/README.md (hunk 1 above)
Comment (tstavinoha) [reply third-party/README.md]: That is a great suggestion, it describes the extension much better. Thanks!

**Comment by chanseokoh** [comment]
Thanks a lot! This is awesome.

**Pull request merged**